source/_posts/2021-11-12-100k-analytics.markdown

```diff
@@ -0,0 +1,122 @@
+
+The 100,000 installations are not the total number of Home Assistant users. The truth here is that we don't know the total number of installations because Home Assistant is private by design. Our estimate is that there are 4-5x more installations than people that opt-in to analytics.
+
+_For the latest version of the graphs in this post, visit [Home Assistant Analytics][analytics]_
+
+![Line graph showing active installations over time hitting 100000 installations.](/images/blog/2021-11-analytics/active-installations.png)
+
+<!--more-->
+
+## Why we collect data
+
+It's good to start off with why we allow users to opt-in to share some data with us.
+
+Manufacturers of IoT products live in their own world in which we have little presence. We don't have stands at CES or other trade shows. Instead, manufacturers see we're open source and assume it's a small user base. They assume the ability to integrate with Home Assistant and other platforms is not profitable enough to allocate resources to add the necessary APIs to integrate.
+
+With the data collected by Home Assistant Analytics we can show that providing an API can expand their market around the globe.
+
+<p class='img'>
+<img src='/images/blog/2021-11-analytics/onboarding-analytics.png'>
+Screenshot of analytics during Home Assistant onboarding
+</p>
+
+## How people run Home Assistant
+
+The collected data is also used to give us, the Home Assistant developers, insight in how Home Assistant is installed and what we should improve. As you can see in our installation breakdown, the operating system installation type is by far the most popular. This makes sense because it is our recommended installation type and the easiest to use. It gives the full Home Assistant experience and can be fully managed via the user interface. No text editors, Linux experience or computer science degree necessary.
+
+![Pie chart of installation types. Operating System has 66.9%. Container has 17.9%. Supervised has 10.2%. Core has 5%.](/images/blog/2021-11-analytics/installation-types.png)
+
+The board breakdown below only applies to the operating system installation type. The Raspberry Pi family of devices is by far the most popular way of running Home Assistant, followed by running it in a virtual machine. Running Home Assistant in a virtual machine is popular because it is our recommended approach for people that already have a server at home.
+
+![Pie chart showing what boards are used. Raspberry Pi 4 has 35.2%. Virtual Machine has 32.9%.](/images/blog/2021-11-analytics/board-breakdown.png)
+
+## Tracking updates
+
+We have spent a lot of time on making sure that it is easy to update both Home Assistant Core and Home Assistant Operating System. Ease of updates are important because when critical bugs or security issues arise, users should feel confident to hit the update button.
```

Comment:
```suggestion
We have spent a lot of time making sure that it is easy to update both Home Assistant Core and Home Assistant Operating System. Ease of updates is important because when critical bugs or security issues arise, users should feel confident to hit the update button.
```

Reply:
I think it's the ease that is important not the updates. So this should have been singular.

Reply:
Nevermind, I was a bit colorblind here.